extensions/notebook/src/integrationTest/notebookIntegration.test.ts

```diff
@@ -35,12 +35,22 @@ describe('Notebook Integration Test', function (): void {
 		nbformat_minor: 2
 	};
 
+	let installComplete = false;
+	before(async function () {
+		// Ensure Jupyter dependencies are installed first
+		let jupterControllerExports = vscode.extensions.getExtension('Microsoft.notebook').exports;
+		let jupyterController = jupterControllerExports.getJupyterController() as JupyterController;
```

Comment:
Shouldn't we have to await activation here? I think it's possibly better if we are returning a`Promise<IExtensionApi>` that we wait on it somewhere? Very interesting how this works...